lib/neo4j/migrations/helpers.rb

```diff
@@ -0,0 +1,50 @@
+module Neo4j
+  module Migrations
+    module Helpers
+      def rename_property(label, old_property, new_property)
+        by_label(label).where(n: {new_property => nil})
+                       .set("n.#{new_property} = n.#{old_property}")
+                       .remove("n.#{old_property}")
+                       .exec
+      end
+
+      def drop_nodes(label)
+        query.match("(n:`#{label}`)")
```

Comment:
You can also do: `query.match(n: label)`, I think.

Kudos on not going with the `DETACH DELETE` (though of course there's no method for it.  Just put a note on my list to add one ;)